pagination.js

```diff
@@ -1,3 +1,4 @@
+/*eslint-env jquery*/
```

Comment:
This shouldn't be necessary if you define it in the `env` in `.eslintrc.js`, which you do.  Or am I missing something?

Reply:
My apologies, you're I was having some issues with the command earlier on my end because my NodeJs was out of date. I added those lines experimentally , I'll proceed to push a cleaner version.